packages/react/src/views/ChatBody/ChatBody.styles.js

```diff
@@ -9,7 +9,7 @@ export const getChatbodyStyles = () => {
       overflow-x: hidden;
       display: flex;
       flex-direction: column-reverse;
-      max-height: 600px;
+      max-height: 900px;
```

Comment:
So, even after this PR, when this particular element exceeds the limit, the behavior will still be the same, right?

Can’t we consider using percentages or relative units instead of fixed units?

Because even with this change, there doesn't seem to be any real benefit.

What’s your opinion on this?

Reply:
Ok , I am changing it to 100%